I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -411,13 +409,13 @@
    for a WDM TE-link between WDM-TE-nodes (OMS MCG). In this scenario,
    the node containing the remote optical transponders can be
    considered as WDM-TE-node with termination capability only and no
-   no switching capabilities.
+   switching capabilities.
   </t>
   
   <t>
-   An OMS MCG is terminated on both ends by a link termination point
+   An WDM TE-link is terminated on both ends by a link termination point
    (LTP) as defined in <xref target="RFC8345"/>.
-   Links in optical transport networks are typically bidirectional but
+   Links in optical transport networks are typically bidirectional but can have different impairments in the two directions. Therefore they 
```

Comment:
Weekly call Febryuary 21: 
change links into TE-links

Reply:
Checked the context and here is a text proposal how the paragraph could be updated:
```
   A WDM TE-link is terminated on both ends by a link termination point
   (LTP) as defined in <xref target="RFC8345"/>.
   Links between WDM nodes in optical transport networks are typically
   bidirectional.
   Generally, they have different impairments in the two directions and
   hence they have to be modeled as a pair of two unidirectional
   TE-links following the  <xref target="RFC8345"/> modeling approach.
```

Reply:
> Checked the context and here is a text proposal how the paragraph could be updated:
> 
> ```
>    A WDM TE-link is terminated on both ends by a link termination point
>    (LTP) as defined in <xref target="RFC8345"/>.
>    Links between WDM nodes in optical transport networks are typically
>    bidirectional.
>    Generally, they have different impairments in the two directions and
>    hence they have to be modeled as a pair of two unidirectional
>    TE-links following the  <xref target="RFC8345"/> modeling approach.
> ```

I would add RFC8795 as referecne , together with RFC8345, since TE-link is defined there and not in RFC8345

Reply:
Reference to RFC8345 replaced with reference to RFC8795, which defines the TE-link and the LTP.

```
   A WDM TE-link is terminated on both ends by a link termination point
   (LTP) as defined in [RFC8795].  Links between WDM nodes in optical
   transport networks are typically bidirectional.  Generally, they have
   different impairments in the two directions and hence they have to be
   modeled as a pair of two unidirectional TE-links following the
   [RFC8795] modeling approach.
```
This update will be part of the next commit.